### DISCUSSION TITLE
Deciding the scope of Pandit Project with Respect to Manuscript Collections

### DISCUSSION THREAD

**Discussion by ybronner** [PanditProject/Website#19] [Help]
A topic that came up apropos other discussions prompted me to ask how far should we extend the mission of Pandit, a prosopographical database, to help people locate manuscripts or scans and other copies thereof (a bibliographical mission). On the one hand, helping people access manuscripts is not our mandate (and, in some cases, makes the data structure overly complex). On the other, sometimes  information on where to get a scan of a manuscript is hugely important given the politics of manuscript accessibility. 
I'm eager for input.

**Comment by petersonjonath** [comment]
To my mind, PANDiT fulfills a different function than bibliographic tools like the NCC, individual print catalogues, library sources (WorldCat, etc.). As a prosopographical tool, Pandit might occasionally use bibliographic points to plot connections and associations, but its mandate, so far as I've understood it, is not to provide a comprehensive bibliographic account. The existing bibliographic data seems in many ways to be a hold over of the Potter encyclopedias, and other data sets. I'd assume to express more comprehensive bibliographical data would require a great deal of restructuring, including merging vast new data sets. Yigal's question, while posed in the context of PANDiT, raises the larger issue of what, precisely, prosopography in a South Asian context should do. I suppose it's up to us to decide. The kind of associations bibliographic data expresses takes us astray, in most cases, of prosopography as the study of "persona-writing." In a S.A. context, I take the study of "persona-writing" to mean centering the writer, their family associations, their preceptorial connections, and so on–––all of the rich associations that PANDiT already accommodates.

**Comment by wujastyk** [comment]
I agree with both of you.  I think PP could benefit by backing away from detailed manuscript cataloguing and instead developing in the direction of what Jonathan calls "persona-writing".  

Having said that, I think there's no harm in giving, for example, giving MS information (for the reasons you give, Yigal), and also a link to a MS scan, if it exists, as part of the MS description.  But we could keep that very simple and basic.  No need to qualify the MS in detail. Date, script no. of folios and location is probably plenty.  A note field could carry anything essential beyond that.  

About linking to scans, maybe the way forward with such entities is to give a generic field for the target entity (e.g., a URL) and then a tweet-length note field for the user to explain briefly anything essential.  This reminds me of the data design of [PhiloBiblon](https://bancroft.berkeley.edu/philobiblon/history_en.html), and I think it was a strength.  Each data field had two associated fields: "quality" where one could express the level of trust for the data, and "basis" for giving the source of the information.  Like [this](https://bancroft.berkeley.edu/philobiblon/windowsversion_en.html):
![image](https://user-images.githubusercontent.com/762246/125681448-60bf6c72-9c39-4f82-92e4-d583b7a9e453.png)

I'm not suggesting PP imitate this globally, but the concept could help in the context of fields that contain useful data but that aren't mission-central to PP. 

I am worried about the drive in PP at the moment to focus on more and more detail about manuscript measurements, foliation, types of paper, and so forth.  I would rather see developments in the direction of displaying family and lineage trees graphically (like [this](https://jaina-prosopography.org/network/person/492)), or being able to export a snapshot of the manuscripts of a work in a manner that would form part of the introduction to a critical edition.   In other words, persona-writing and tools for active manuscript editors.
